cmd/blockchaincmd/change_weight.go

```diff
@@ -189,7 +194,15 @@ func setWeight(_ *cobra.Command, args []string) error {
 			return err
 		}
 
-		allowedChange := float64(totalWeight) * constants.MaxL1TotalWeightChange
+		churnSettings, err := validatormanagersdk.GetChurnSettings(
+			validatorManagerRPCEndpoint,
+			common.HexToAddress(validatorManagerAddress),
+		)
+		if err != nil {
+			return err
+		}
+
+		allowedChange := float64(totalWeight*uint64(churnSettings.MaximumChurnPercentage)) / 100.0
```

Comment:
make this into a function since it was also used in add validator?

Reply:
the function is useful itself, but not useful in change weight because the logic use the intermediate variables
more than once. but adding a function just to be used in add validator and as general utility on SDK